### PR TITLE
Added allele field to NetMHCstab output

### PR DESCRIPTION
**Pull request by agraubert**
For unsupported alleles, NetMHCStab picks the closest supported allele

This new field in the output gives the allele used in predictions for each entry

### CODE REVIEW

File: pvacseq/lib/netmhc_stab.py [hunk 1]
```diff
@@ -27,10 +27,11 @@ def main(args_input = sys.argv[1:]):
     jobid_searcher = re.compile(r'<!-- jobid: [0-9a-fA-F]*? status: (queued|active)')
     result_delimiter = re.compile(r'-{20,}')
     fail_searcher = re.compile(r'(Failed run|Problematic input:)')
+    allele_searcher = re.compile(r'^(.*?) : Distance to traning data .*? nearest neighbor (.*?)\)$', re.MULTILINE)
```
Comment (susannasiebert) [inline pvacseq/lib/netmhc_stab.py]: There is a typo here: `traning`.

**Comment by susannasiebert** [comment]
+1 once the merge conflict is resolved.

File: pvacseq/lib/netmhc_stab.py (hunk 1 above)
Comment (agraubert) [inline pvacseq/lib/netmhc_stab.py]: Yeah it took me forever to figure out why the correctly spelled regex wasn't working

**Comment by agraubert** [comment]
I updated that regex now.  If they ever fix the typo, it'll still match

**Pull request merged**